dts/bindings/led_strip/worldsemi,ws2812-gpio.yaml

```diff
@@ -7,7 +7,16 @@ description: |
   Driver bindings for bit-banging a WS2812 or compatible LED strip.
 
   The GPIO driver uses inline assembly, and isn't available for all
-  boards.
+  boards. The timing used by the driver can be changed by providing
```

Comment:
In this case, It is preferable to set the Kconfig value directly.

Reply:
You're correct, I'll rephrase. Something like: "The timing is automatically derived from the cpu clock frequency, or can be manually provided by the Kconfig settings DELAY_Txx."?

Reply:
It's no problem.
If you change it as above, the microbit_bbc example is unsuitable.
It also needs to be fixed. (Create microbit_bbc.conf.)